tests/aws/services/cloudformation/resources/test_route53.py

```diff
@@ -44,3 +44,15 @@ def test_create_record_set_without_resource_record(deploy_cfn_template, route53_
         ),
         parameters=parameters,
     )
+
+
+@markers.aws.validated
+def test_create_health_check(
+    deploy_cfn_template,
+):
+    deploy_cfn_template(
+        template_path=os.path.join(
+            os.path.dirname(__file__),
+            "../../../templates/route53_healthcheck.yml",
+        ),
+    )
```

Comment:
nit: would be great if you could add a service call here to verify deployment (& ideally a snapshot of it)